convene-web/app/lib/furniture.rb

```diff
@@ -1,9 +1,16 @@
 module Furniture
   REGISTRY = {
-    breakout_tables: Furniture::BreakoutTables,
-    videobridge_jitsi: Furniture::VideobridgeJitsi,
+    breakout_tables: Furniture::BreakoutTablesByJitsi,
```

Comment:
Minor: we could consider renaming the symbol as well, to include "jitsi" in the name.

Reply:
Done!